blazor/datagrid/filtering.md

```diff
@@ -461,15 +461,63 @@ Operator |Description |Supported Types
 StartsWith |Checks whether the value begins with the specified value. |String
 EndsWith |Checks whether the value ends with the specified value. |String
 Contains |Checks whether the value contains the specified value. |String
+DoesNotStartWith |Checks whether the value does not begin with the specified value. |String
+DoesNotEndWith |Checks whether the value does not end with the specified value. |String
+DoesNotContain |Checks whether the value does not contain the specified value. |String
 Equal |Checks whether the value is equal to the specified value. |String &#124; Number &#124; Boolean &#124; Date
 NotEqual |Checks for values not equal to the specified value. |String &#124; Number &#124; Boolean &#124; Date
 GreaterThan |Checks whether the value is greater than the specified value. |Number &#124; Date
 GreaterThanOrEqual |Checks whether a value is greater than or equal to the specified value. |Number &#124; Date
 LessThan |Checks whether the value is less than the specified value. |Number &#124; Date
 LessThanOrEqual |Checks whether the value is less than or equal to the specified value. |Number &#124; Date
+IsNull |Returns the values that are null. |String &#124; Number &#124; Date
```

Comment:
Change the order as like we displayed in the operators dropdown.

Reply:
Change as per guidelines.